src/node/rpc/member_frontend.h

```diff
@@ -523,6 +735,45 @@ namespace ccf
            users_view->put(parsed.user_id, user_info.value());
            return true;
          }},
+        {"set_jwt_issuer",
+         [this](ObjectId proposal_id, kv::Tx& tx, const nlohmann::json& args) {
+           const auto parsed = args.get<SetJwtIssuer>();
+           auto issuers = tx.get_view(this->network.jwt_issuers);
+
+           issuers->put(parsed.issuer, parsed);
```

Comment:
Following the other comment on proposal failure, should this be moved after the call to `set_jwt_public_signing_keys()`?

Reply:
I missed that... though it's a bit more tricky here to refactor the code. Not impossible though.

Reply:
Done.